diff --git a/node_modules/xlsx-style/dist/cpexcel.js b/node_modules/xlsx-style/dist/cpexcel.js
index f821c7a..cbb06be 100644
--- a/node_modules/xlsx-style/dist/cpexcel.js
+++ b/node_modules/xlsx-style/dist/cpexcel.js
@@ -804,7 +804,8 @@ if (typeof module !== 'undefined' && module.exports) module.exports = cptable;
   "use strict";
   if(typeof cptable === "undefined") {
     if(typeof require !== "undefined"){
-      var cpt = require('./cpt' + 'able');
+      // var cpt = require('./cpt' + 'able');
+      var cpt = cptable;
       if (typeof module !== 'undefined' && module.exports) module.exports = factory(cpt);
       else root.cptable = factory(cpt);
     } else throw new Error("cptable not found");
diff --git a/node_modules/xlsx-style/xlsx.js b/node_modules/xlsx-style/xlsx.js
index 15b58b1..3c81d8a 100644
--- a/node_modules/xlsx-style/xlsx.js
+++ b/node_modules/xlsx-style/xlsx.js
@@ -1331,6 +1331,8 @@ function getzipdata(zip, file, safe) {
 	try { return getzipdata(zip, file); } catch(e) { return null; }
 }
 
+var JSZip = require('./jszip');
+
 var _fs, jszip;
 if(typeof JSZip !== 'undefined') jszip = JSZip;
 if (typeof exports !== 'undefined') {
